sno/structure.py

```diff
@@ -275,6 +275,9 @@ def instantiate(cls, tree, path, version):
         if not isinstance(tree, pygit2.Tree):
             raise TypeError(f"Expected Tree object, got {type(tree)}")
 
+        if not ".sno-table" in tree:
```

Comment:
```suggestion
        if ".sno-table" not in tree:
```
my linter would dislike this immensely :)

Reply:
Done